Assets/Mirror/Editor/Weaver/Processors/TargetRpcProcessor.cs

```diff
@@ -17,7 +17,7 @@ public static bool HasNetworkConnectionParameter(MethodDefinition md)
 
         public static MethodDefinition ProcessTargetRpcInvoke(TypeDefinition td, MethodDefinition md)
         {
-            MethodDefinition rpc = new MethodDefinition(RpcProcessor.k_RpcPrefix + md.Name, MethodAttributes.Family |
+            MethodDefinition rpc = new MethodDefinition(RpcProcessor.RpcPrefix + md.Name, MethodAttributes.Family |
```

Comment:
Something interesting is going on here, i just left it alone though. Why is there Rpc stuff in TargetRpc?